loading training instances, dir instances/randomip_n14_m38 idx 45
Set parameter Username
Academic license - for non-commercial use only - expires 2023-06-01
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
[0]
/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/cvxpy/expressions/expression.py:593: UserWarning:
This use of ``*`` has resulted in matrix multiplication.
Using ``*`` for matrix multiplication has been deprecated since CVXPY 1.1.
    Use ``*`` for matrix-scalar and vector-scalar multiplication.
    Use ``@`` for matrix-matrix and matrix-vector multiplication.
    Use ``multiply`` for elementwise multiplication.
This code path has been hit 1 times so far.
  warnings.warn(msg, UserWarning)
Traceback (most recent call last):
  File "/Users/zhangqiming/Desktop/RL_research/Reinforcement-Learning/project/exmaple.py", line 205, in <module>
    compute_solution()
  File "/Users/zhangqiming/Desktop/RL_research/Reinforcement-Learning/project/exmaple.py", line 162, in compute_solution
    INtsolve(A, b, c)
  File "/Users/zhangqiming/Desktop/RL_research/Reinforcement-Learning/project/exmaple.py", line 13, in INtsolve
    prob = cp.Problem(cp.Maximize(c * x),
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/cvxpy/expressions/expression.py", line 717, in __array_ufunc__
    return ufunc_handler(self, args[0])
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/cvxpy/expressions/expression.py", line 72, in <lambda>
    np.multiply: lambda self, a: self.__rmul__(a),
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/cvxpy/expressions/expression.py", line 49, in cast_op
    return binary_op(self, other)
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/cvxpy/expressions/expression.py", line 639, in __rmul__
    return other * self
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/cvxpy/expressions/expression.py", line 49, in cast_op
    return binary_op(self, other)
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/cvxpy/expressions/expression.py", line 596, in __mul__
    return cvxtypes.matmul_expr()(self, other)
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/cvxpy/atoms/affine/binary_operators.py", line 50, in __init__
    super(BinaryOperator, self).__init__(lh_exp, rh_exp)
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/cvxpy/atoms/atom.py", line 51, in __init__
    self._shape = self.shape_from_args()
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/cvxpy/atoms/affine/binary_operators.py", line 120, in shape_from_args
    return u.shape.mul_shapes(self.args[0].shape, self.args[1].shape)
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/cvxpy/utilities/shape.py", line 142, in mul_shapes
    lh_shape, rh_shape, shape = mul_shapes_promote(lh_shape, rh_shape)
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/cvxpy/utilities/shape.py", line 109, in mul_shapes_promote
    lh_shape, rh_shape))
ValueError: Incompatible dimensions (1, 14) (8, 1)
[0]
[0]
[0]
[0]
[0]
[0]
size of A :  (38, 14)
size of B :  (38,)
size of c :  (1, 14)